### ISSUE TITLE
Richer interaction with brokered RDS instances

### ISSUE THREAD

**Issue by mogul** [18F/cg-product#586]
What we're after: Users are able to configure RDS read-replicas, get metrics, and send alerts without having their own AWS access

Benefits/why:

- Enable fine-grained management of the brokered RDS service
- New features for all users of RDS
- Unblock full FEC migration into cloud.gov

Notes:

- In scope:  

  - Set read-replica count via broker parameter
  - Get a read-only IAM role via service-key suitable for getting metrics

- Out of scope:
  - Alerting based on metrics is left to users (though we can walk FEC through using OpenDuty)

Metrics we could track/target:

- Number of migrations blocked by lack of RDS features

**Comment by mogul** [comment]
@afeld can you parent any existing stories about this topic onto this feature?

**Comment by afeld** [comment]
@mogul Added!
